### PR TITLE
mdd-seo-stacked-272382-272385

### PR DESCRIPTION
**Pull request by mddifilippo89**
seo - 272382-272385

Stacked:
Spline
SplineArea
Line
Area

### CODE REVIEW

File: samples/charts/data-chart/stacked-area-chart/App.razor [hunk 1]
```diff
@@ -0,0 +1,50 @@
+﻿@page "/StackedAreaChart"
```
Comment (HUSSAR-mtrela) [inline samples/charts/data-chart/stacked-area-chart/App.razor:1]: remove this line

File: samples/charts/data-chart/stacked-line-chart/App.razor [hunk 2]
```diff
@@ -0,0 +1,43 @@
+﻿@page "/StackedLineChart"
```
Comment (HUSSAR-mtrela) [inline samples/charts/data-chart/stacked-line-chart/App.razor:1]: remove this line

File: samples/charts/data-chart/stacked-spline-area-chart/App.razor [hunk 3]
```diff
@@ -0,0 +1,43 @@
+﻿@page "/StackedSplineAreaChart"
```
Comment (HUSSAR-mtrela) [inline samples/charts/data-chart/stacked-spline-area-chart/App.razor:1]: remove this line

File: samples/charts/data-chart/stacked-spline-chart/App.razor [hunk 4]
```diff
@@ -0,0 +1,43 @@
+﻿@page "/StackedSplineChart"
```
Comment (HUSSAR-mtrela) [inline samples/charts/data-chart/stacked-spline-chart/App.razor:1]: remove this line

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: samples/charts/data-chart/stacked-area-chart/App.razor (hunk 1 above)
Comment (mddifilippo89) [reply samples/charts/data-chart/stacked-area-chart/App.razor:1]: resolved, pending

File: samples/charts/data-chart/stacked-line-chart/App.razor (hunk 2 above)
Comment (mddifilippo89) [reply samples/charts/data-chart/stacked-line-chart/App.razor:1]: resolved, pending

File: samples/charts/data-chart/stacked-spline-area-chart/App.razor (hunk 3 above)
Comment (mddifilippo89) [reply samples/charts/data-chart/stacked-spline-area-chart/App.razor:1]: resolved, pending

File: samples/charts/data-chart/stacked-spline-chart/App.razor (hunk 4 above)
Comment (mddifilippo89) [reply samples/charts/data-chart/stacked-spline-chart/App.razor:1]: resolved, pending

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**